src/_layouts/none.liquid

```diff
@@ -0,0 +1 @@
+{{content}}
```

Comment:
Since 11ty is a lot less opinionated, it treats a lot more files like templates. Since the default layout is the normal page setup, a bunch of files that shouldn't end up as pages were being treated as other pages. This was the easiest way to add a layout for files that aren't pages.

I'm hoping to find a better way to configure this.